contrib/ibmock/efa.c

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
+#include <infiniband/efadv.h>
+#include <infiniband/verbs.h>
+
+#include <errno.h>
+#include <stdlib.h>
+
+#include "fake.h"
+#include "verbs.h"
+#include "config.h"
+
+int efadv_query_device(struct ibv_context *context,
+                       struct efadv_device_attr *attr, uint32_t inlen)
+{
+    if (!context || inlen != sizeof(efa_dev_attr)) {
```

Comment:
IIRC we need != NULL and brackets around both conditions

Reply:
thanks, tried to fix for all occurences